server/helpers/commonHelper.js

```diff
@@ -0,0 +1,120 @@
+const map = require('lodash/map');
+const includes = require('lodash/includes');
+const head = require('lodash/head');
+const split = require('lodash/split');
+const keys = require('lodash/keys');
+const groupBy = require('lodash/groupBy');
+const get = require('lodash/get');
+const concat = require('lodash/concat');
+
+const { getProjects } = require('./testrailReportHelper');
+
+const { parseTestCase, extractTestCase } = require('./parserHelper');
+const { createSection, getProjectRepoSections } = require('./sectionHelper');
+const { getProjectRepoCases, createSuiteCases } = require('./testCaseHelper');
+const logger = require('../utils/logger');
+const {
+  createTestSuite,
+  getSuiteName,
+  getProjectSuites,
+} = require('./testSuiteHelper');
+
+const getParsedTestCase = data => {
+  const { content, testName } = data;
+  const testCaseBlock = extractTestCase(content, 'test_case');
+
+  if (!testCaseBlock) {
+    logger.log('warn', `TEST CASE FOR IT BLOCK '${testName}' NOT FOUND`);
+    return null;
+  }
+
+  return parseTestCase(testCaseBlock, testName);
+};
+
+const getRepoProjectSuiteNames = data =>
```

Comment:
put common suites and common sections and common testcases in each project in the repo